spec/amp-cache-debugging.md

```diff
@@ -0,0 +1,23 @@
+# AMP Cache Debugging
+
+**Why is my doc broken on an AMP cache?**
+
+Valid AMP documents typically appear and behave the same on AMP Caches as they
+do on the origin. However, there are some components and server configurations
+that can be problematic.
+
+If a particular document appears and behaves as expected on your origin, but not
+when viewed via the cache (how to map origin URLs to [Google's AMP Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)),
+first open the developer console in your browser, and ensure no errors or
```

Comment:
I think this would read better if the "steps" were listed as bullets, so that it's easy to parse the info, like so:

If a particular document appears and behaves as expected on your origin, but not when viewed via the cache (how to map origin URLs to [Google's AMP Cache](https://developers.google.com/amp/cache/overview#amp-cache-url-format)), try the following:

- Validate that there are no errors or warnings for the AMP page by reviewing the Chrome DevTools console.
- Validate the structured data and AMP urls for the page by using the AMPBench tool.
- If you still have a problem, review the table below: